src/store/actions/playerActions.js

```diff
@@ -25,3 +28,35 @@ export const getSongStreamLink = (path) => (dispatch, getState) => {
       throw new Error(err);
     });
 };
+
+export const playNextSong = () => (dispatch, getState) => {
+  const songIndex = getState().songsQueue.get('index');
+  const songsQueue = getState().songsQueue.get('queue');
+
+  const canPlayNextSong = songIndex + 1 <= songsQueue.size;
+
+  if (canPlayNextSong) {
+    const nextIndex = songIndex + 1;
+    // eslint-disable-next-line no-underscore-dangle
+    const nextSongPath = songsQueue.get(nextIndex)._root.entries[2][1];
```

Comment:
?

Reply:
You shouldn't use `_root`. Check this doc for a ref to all methods available when working with lists: `https://immutable-js.github.io/immutable-js/docs/#/List`

Reply:
Okay, i'll check it out 😄